DataStructures/Dictionaries/OpenAddressingHashTable.cs

```diff
@@ -0,0 +1,225 @@
+﻿/***
+ * Open Addressing Hash Table
+ * 
+ * Implementation of Open Addressing Hash Table using
+ *      - Linear Probing (coming soon)
+ *      - Quadratic Probing (coming soon)
+ *      - Double Hashing
+ *      
+ * Author: Samuel Kenney
+ * Created: 6/7/2017
+ * Last Modified: 6/13/2017
+ * 
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+using DataStructures.Common;
+using DataStructures.Hashing;
+using System.Threading.Tasks;
+
+namespace DataStructures.Dictionaries
+{
+    /// <summary>
+    /// Open Addressing Data Structure
+    /// </summary>
+    /// <typeparam name="TKey"></typeparam>
+    /// <typeparam name="TValue">will always be an integer since it maps to an index</typeparam>
+    public class OpenAddressingHashTable<TKey>
```

Comment:
Extend the class definition to:
1) Specify TValue types.
2) Implement the `IDictionary<TKey, TValue>` interface.
3) The functionality of hash table should be identical to the functionality of the other Hash Tables in the project in the sense of mapping keys to values.